content/api/resources/operations/object/allow-trust.mdx

```diff
@@ -26,7 +26,7 @@ See the [`Allow Trust` errors](../../../errors/result-codes/operation-specific/a
   - The code for the asset.
 - authorize
   - boolean
```

Comment:
`boolean` -> `int`, right?

Reply:
Fixed!  Other than that, does it check out, @Shaptic?

Reply:
I'd personally prefer to put the possible values of the flag in order, but I do see merit in a "sorted by order of interest" approach, as well. In either case, LGTM! However, I will say I can't actually confirm the _content_ of the docs (that is, I don't know if those numbers for `authorize` match reality).

Reply:
(I initially just hopped into this PR for my own edification.)

Reply:
That's a fair point re: the ordering.  I tinkered with it a bit, tried it one way, then another.  Now I'm reverting back to what I started with, which is also what you're suggesting.